File: Algorithm/Programmers/Level1/solution(32).py
# 완주하지 못한 선수

def solution(participant, completion):
    answer = ''
    # completion은 participant보다 길이가 1 작음
    # 정렬을 통해 같지 않은 것 찾기
    participant = sorted(participant)
    completion = sorted(completion)
    for i in range(len(completion)):
        if participant[i] != completion[i]:
            answer = participant[i]
            return answer
    answer = participant[-1]
    return answer

File: Algorithm/Programmers/Level1/solution(32).java
// 완주하지 못한 선수
// 단 한 명의 선수만 완주하지 못한 상황이므로, 완주 리스트를 기준으로 비교하면 됨

import java.util.*;

class Solution {
    public String solution(String[] participant, String[] completion) {
        String answer = "";
        Arrays.sort(participant);
        Arrays.sort(completion);
        for (int i = 0; i < completion.length; i++){
            if (!(completion[i].equals(participant[i])))
                return participant[i];
        }
        return participant[participant.length-1];
    }
}